src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -111,3 +119,49 @@ inline fun createBitmap(
 ): Bitmap {
     return Bitmap.createBitmap(width, height, config, hasAlpha, colorSpace)
 }
+
+
+/**
+ * Bitmap to bytes.
+ *
+ * @param format The format of bitmap.
+ * @param quality Hint to the compressor, 0-100. 0 meaning compress for small size, 100 meaning compress for max quality.
+ * @return ByteArray
+ */
+fun Bitmap.toBytes(format:CompressFormat = JPEG, quality: Int=100):ByteArray
+        = ByteArrayOutputStream().also { compress(format, quality, it) }.toByteArray()
+
+/**
+ * Return the clipped bitmap.
+ *
+ * @param x      The x coordinate of the first pixel.
+ * @param y      The y coordinate of the first pixel.
+ * @param width  The width.
+ * @param height The height.
+ * @return the clipped bitmap
+ */
+fun Bitmap.clip(x:Int, y:Int, width: Int, height: Int):Bitmap
```

Comment:
I'm not sure this method is necessary. It doesn't save much over the `Bitmap.createBitmap()` call for an operation that might not be very common. At least it should be `inline`, better documented and the return type doesn't need to be specified.

Reply:
`clip` is of more clear meaning than `createBitmap` for bitmap operation. I added `inline` for all methods and updated docs.